Source/WebCore/Modules/mediasource/MediaSource.cpp

```diff
@@ -988,17 +1069,19 @@ bool MediaSource::isTypeSupported(ScriptExecutionContext& context, const String&
     parameters.contentTypesRequiringHardwareSupport = WTFMove(contentTypesRequiringHardwareSupport);
 
     if (RefPtr document = dynamicDowncast<Document>(context)) {
-        auto& settings = document->settings();
-        if (!contentTypeMeetsContainerAndCodecTypeRequirements(contentType, settings.allowedMediaContainerTypes(), settings.allowedMediaCodecTypes()))
+        if (!contentTypeMeetsContainerAndCodecTypeRequirements(contentType, document->settings().allowedMediaContainerTypes(), document->settings().allowedMediaCodecTypes()))
             return false;
```

Comment:
Ditto.

Reply:
Same as above, those preferences are only found in `Settings` not in `SettingsValues` (a Settings class contain a SettingsValues class for storing the numerical value.

`Settings` live in the Document; not a Worker.